fdbserver/TenantCache.actor.cpp

```diff
@@ -51,15 +51,15 @@ class TenantCacheImpl {
 	ACTOR static Future<Void> build(TenantCache* tenantCache) {
 		state Transaction tr(tenantCache->dbcx());
 
-		TraceEvent(SevInfo, "BuildingTenantCache", tenantCache->id()).log();
+		TraceEvent(SevInfo, "TenantCacheBuildStarting", tenantCache->id());
 
 		try {
 			state std::vector<std::pair<int64_t, TenantMapEntry>> tenantList = wait(getTenantList(tenantCache, &tr));
 
 			for (int i = 0; i < tenantList.size(); i++) {
 				tenantCache->insert(tenantList[i].second);
 
-				TraceEvent(SevInfo, "TenantFound", tenantCache->id())
+				TraceEvent(SevDebug, "TenantCacheTenantFound", tenantCache->id())
```

Comment:
Is this trace spammy?

Reply:
We don't test with very large number of tenants in simulation so this shouldn't be an issue. (It probably would've been spammy in prod, hence reducing the severity to SevDebug.)